<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>328</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>271</width>
     <height>241</height>
    </rect>
   </property>
   <property name="title">
    <string>聊天记录</string>
   </property>
   <widget class="QPlainTextEdit" name="plainTextEditMessage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>241</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="lineEditMessage">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>410</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonSend">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>490</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>272</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Port:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditPort"/>
    </item>
    <item>
     <widget class="QPushButton" name="buttonStartServer">
      <property name="text">
       <string>启动服务器</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
